### PR TITLE
Run RuboCop on CI

### PR DESCRIPTION
**Pull request by tido64**

### CODE REVIEW

**Comment by tido64** [comment]
> Didn’t you have this as part of the Gemfile? Because I think you want to lock this so you don’t suddenly get a different version and rules on CI.

Good point!

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -12,6 +12,8 @@ jobs:
       uses: actions/checkout@v2
     - name: minitest
       run: |
+        bundle install
+        rubocop
```
Comment (alloy) [inline .github/workflows/build.yml]: This should probably be prefixed with `bundle exec`, no? Same actually goes for the command below.

Comment (tido64) [reply .github/workflows/build.yml]: Thanks for pointing that out! I'm a total Ruby noob, so this is really helpful.

Comment (alloy) [reply .github/workflows/build.yml]: 🤗

**Pull request merged**